sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts

```diff
@@ -424,61 +425,66 @@ export class ConfirmationOfEnrollmentService {
   ): Promise<UpdateResult> {
     const auditUser = { id: auditUserId } as User;
     const now = new Date();
-    return await this.dataSource.transaction(
-      async (transactionalEntityManager) => {
-        const updateResult = await transactionalEntityManager
-          .getRepository(DisbursementSchedule)
-          .createQueryBuilder()
-          .update(DisbursementSchedule)
-          .set({
+    return this.dataSource.transaction(async (transactionalEntityManager) => {
```

Comment:
Not a blocker or even not part of the PR. `disbursementScheduleId` seems to be obsolete here.

I could see that it is used in the `createNotificationForDisbursementUpdate` but it again uses the disbursement id to get assessment and application and student details.

Reply:
If time allows and not many changes are needed I can take a look on it.